lms/djangoapps/instructor/tests/test_api.py

```diff
@@ -3385,10 +3383,26 @@ def tearDownClass(cls):
 
     def setUp(self):
         super().setUp()
+        test_subject = '\u1234 test subject'
```

Comment:
I moved this from `setUpClass` to the `setUp` function so that the new unit tests testing the scheduling functionality don't poison the test data. Depending on the test we are modifying the `full_test_message` dict, which was causing issues with the older tests.